crates/config/src/toolchain/python_config.rs

```diff
@@ -0,0 +1,36 @@
+// use super::bin_config::BinEntry;
+use schematic::Config;
+use serde::Serialize;
+use version_spec::UnresolvedVersionSpec;
+use warpgate_api::PluginLocator;
+
+#[derive(Clone, Config, Debug, PartialEq, Serialize)]
+pub struct PipConfig {
+    /// List of arguments to append to `pip install` commands.
+    pub install_args: Option<Vec<String>>,
+
+    /// The version of pip to download, install, and run `pip` tasks with.    
+    pub version: Option<UnresolvedVersionSpec>,
+}
+
+#[derive(Clone, Config, Debug, PartialEq)]
+pub struct PythonConfig {
+    /// Location of the WASM plugin to use for Python support.
+    pub plugin: Option<PluginLocator>,
+
+    /// Options for pnpm, when used as a package manager.
+    #[setting(nested)]
+    pub pip: Option<PipConfig>,
+
+    /// The relative root of the virtual environment workspace. Default to moon's
+    /// workspace root
+    #[setting(default = ".", skip)]
+    pub venv_root: String,
```

Comment:
How do venv's work in python? Are there multiple or just one? Or per project? etc.

Reply:
virtual environments can be created per project

But I like the idea to have the single version policy, maybe I am just removing this setting.